IFC4x3/PropertyConstants/l/LRM_RELATIVE_0vbpXQVsr9B98cHX$zd2dr.xml

```diff
@@ -1,5 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="LRM_RELATIVE_0vbpXQVsr9B98cHXzd2dr" Name="LRM_RELATIVE" UniqueId="3997385a-7f6d-492c-9226-461ffd9c29f5">
-	<Documentation>Measurements are made from the specified “from referent” location, in the direction of the linear element.</Documentation>
+	<Documentation>Measurements are made from the specified “from referent” location, in the direction of the linear element.
```

Comment:
Where is the direction of the linear element defined?

Reply:
We'll have to look into this one. Since we don't have a "from referent", maybe this can also be deleted?